src/__tests__/sigstore.test.ts

```diff
@@ -30,7 +31,6 @@ import { trustedRoot } from './__fixtures__/trust';
 import type { VerifyOptions } from '../config';
 
 jest.mock('../sign');
-jest.mock('../tuf');
```

Comment:
Instead of mocking out the entire `tuf` module, we're using the new `@tufjs/repo-mock` package to mock TUF at the API layer. This gives us better coverage of the TUF code paths while still avoiding traffic to the actual TUF repository.

Reply:
Very nice!